src/utils/__tests__/get-sorted-nodes-by-import-order.spec.ts

```diff
@@ -187,11 +190,12 @@ test('it returns all sorted nodes with sort order case-insensitive', () => {
     const result = getImportNodes(code);
     const sorted = getSortedNodesByImportOrder(result, {
         importOrder: ['^a$', '^t$', '^k$', '^B', '^[./]'],
+        importOrderBuiltinModulesToTop: false,
         importOrderCaseInsensitive: true,
-        importOrderSeparation: false,
         importOrderGroupNamespaceSpecifiers: false,
+        importOrderMergeDuplicateImports: false,
+        importOrderSeparation: false,
```

Comment:
I prefer not making formatting-only changes like alphabetizing as part of a large PR like this, it adds noise.  Let's make small tweaks like this (especially when there's a lot of them) in a separate PR.  But don't undo it here.